central/globaldb/postgres.go

```diff
@@ -111,7 +111,9 @@ func InitializePostgres(ctx context.Context) *postgres.DB {
 		})); err != nil {
 			log.Fatalf("Timed out trying to open database: %v", err)
 		}
-
+		if _, err := postgresDB.Exec(context.Background(), "alter database central_active set synchronous_commit = on"); err != nil {
```

Comment:
should it be `synchronous_commit = off`?

Reply:
No, this is setting async commit on for normal Central running

Reply:
ah, then the error message below this line should probably be updated.